### PR TITLE
TouchableHighlight & TouchableOpacity: add react-native-web 0.16 View props

### PR DESCRIPTION
**Pull request by jfrolich**

### CODE REVIEW

File: src/components/TouchableHighlight.res [hunk 1]
```diff
@@ -43,4 +43,14 @@ external make: (
   ~testID: string=?,
   ~touchSoundDisabled: bool=?,
   ~children: React.element=?,
+  // react-native-web 0.16 View props
```
Comment (MoOx) [inline src/components/TouchableHighlight.res:46]:
```suggestion
  // react-native-web 0.16 Touchable (View) props
```

File: src/components/TouchableOpacity.res [hunk 2]
```diff
@@ -40,4 +40,14 @@ external make: (
   ~testID: string=?,
   ~touchSoundDisabled: bool=?,
   ~children: React.element=?,
+  // react-native-web 0.16 View props
```
Comment (MoOx) [inline src/components/TouchableOpacity.res:43]:
```suggestion
  // react-native-web 0.16 Touchable (View) props
```

**Review by MoOx — APPROVED** [review]
What do you think about having more explicit comment for futur maintenance?

**Comment by MoOx** [comment]
((I hesitated to add them. Touchable* might be deprecated (in favour of Pressable (doc says: _"If you're looking for a more extensive and future-proof way to handle touch-based input, check out the Pressable API."_) soon and they doesn't really use inherit View props.))

**Comment by jfrolich** [comment]
I don't think they are going to deprecate them very soon because there are so many react-native apps relying on it (including our app that still have a lot of pressables).

**Pull request merged**